Ports/bash/patches/0002-Remove-getconf-from-things-to-build.patch

```diff
@@ -0,0 +1,1355 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Fabian Dellwing <fabian.dellwing@gmail.com>
+Date: Mon, 24 Apr 2023 16:00:48 +0200
+Subject: [PATCH] Remove getconf from things to build
+
+---
```

Comment:
Please note down in the patch description  _why_ we are excluding `getconf`.

In this case, as far as I understand, that is a lack of `libintl.h` and/or us failing to find it when cross-compiling.